src/components/common/validation/time-format/ValidatedTimeEntry.jsx

```diff
@@ -63,14 +29,7 @@ const ValidatedTimeEntry = ({
       autoComplete="off"
       type="text"
       {...register(name, {
-        onChange: () => setFinishTimeText(),
-        required: {
-          value: isRequired,
-          message: errorMessage,
-        },
-        validate: {
-          validateTimeEntry: (value) => isTimeValid(value) || errorMessage,
-        },
+        onBlur: () => updateFinishTimeText(timeType),
```

Comment:
Previously we moved away from onBlur due to not being able to tab between input boxes within a timecard. 
I know we have discussed that we shouldn't be constantly re-rendering the page on every input.
Just wanted to check we can still tab between input boxes.

Reply:
We have moved back to on change due to issues with onBlur